packages/datetime/test/dateRangePickerTests.tsx

```diff
@@ -16,7 +16,7 @@ import * as Errors from "../src/common/errors";
 import { Months } from "../src/common/months";
 import { Classes as DateClasses, DateRange, DateRangePicker, IDateRangePickerProps } from "../src/index";
 
-describe("<DateRangePicker>", () => {
+xdescribe("<DateRangePicker>", () => {
```

Comment:
i prefer `describe.skip` for readability. also we have lint rules banning `describe.skip` so you won't accidentally commit it.